project.clj

```diff
@@ -3,16 +3,17 @@
   :url "https://github.com/derwolfe/fernet-clj"
   :license {:name "MIT"
             :url "https://raw.github.com/derwolfe/fernet-clj/master/LICENSE"}
-  :dependencies [[org.clojure/clojure "1.5.1"]
-                 [org.bouncycastle/bcprov-jdk15on "1.49"]
-                 [commons-codec/commons-codec "1.8"]
-                 [clojurewerkz/buffy "0.3.0"]]
+  :dependencies [[org.clojure/clojure "1.8.0"]
+                 [org.bouncycastle/bcprov-jdk15on "1.54"]
+                 [commons-codec/commons-codec "1.10"]
+                 [clojurewerkz/buffy "1.0.2"]]
   :profiles {:dev {:resource-paths ["test/resources"]
-                   :dependencies [[org.clojure/data.json "0.2.3"]
-                                  [clj-time "0.6.0"]
-                                  [perforate "0.3.3"]]
+                   :dependencies [[org.clojure/data.json "0.2.6"]
+                                  [clj-time "0.11.0"]
```

Comment:
It looks like this is only being used a dev dependency because `clj-time` is only being used in the tests. The TTL apis are provided by using `(int (/ (System/currentTimeMillis) 1000)))` and addition with the TTL (this happens inside of `fernet.core/check-ttl`.